cmd/govim/highlight.go

```diff
@@ -51,7 +51,7 @@ func (v *vimstate) textpropDefine() error {
 }
 
 func (v *vimstate) redefineHighlights(diags []types.Diagnostic, force bool) error {
-	if !force && (v.config.HighlightDiagnostics == nil || !*v.config.HighlightDiagnostics) {
+	if v.config.HighlightDiagnostics == nil || !*v.config.HighlightDiagnostics {
```

Comment:
It bothers me that `force` now means something different for quickfix, signs and highlights. Worth fixing now, or would you like to leave that for a later PR?

Reply:
As per our slack conversation, I'll leave it as is for now.

When this is merged we'll have the following meaning of `force`:
- Highlights: Update even if there are no new diagnostics
- Signs: Update even if there are no new diagnostics
- Quickfix: Update even if there are no new diagnostics AND ignore if auto quickfix is disabled via config